lib/tool_shed/test/base/driver.py

```diff
@@ -45,9 +38,9 @@
 
 
 class ToolShedTestDriver(driver_util.TestDriver):
-    """Instantial a Galaxy-style nose TestDriver for testing the tool shed."""
+    """Instantiate a Galaxy-style TestDriver for testing the tool shed."""
 
-    def setup(self):
+    def setup(self, config_object=None):
```

Comment:
Doesn't seem to be used?

```suggestion
    def setup(self):
```